configs/platforms/win-2012-x64.rb

```diff
@@ -0,0 +1,19 @@
+platform "win-2012-x64" do |plat|
+  plat.vmpooler_template "win-2012r2-x86_64.make"
+
```

Comment:
you can define the platform triple here and in the x86 one below instead of in the project config.